factfinder/main.py

```diff
@@ -196,13 +333,16 @@ def aggregate_horizontal(self, client, v, geotype) -> pd.DataFrame:
         M_variables = (
             [i + "M" for i in v.census_variable] if v.source != "decennial" else []
         )
-        census_variables = E_variables + M_variables
-        df = self.download_variable(client, census_variables, geotype)
+        return E_variables, M_variables
+
+    def aggregate_horizontal(self, client, v:Variable, geotype:str) -> pd.DataFrame:
+        """
+        this function will aggregate multiple census_variables into 1 pff_variable
```

Comment:
Maybe we should clarify in the function description that the aggregation here is strictly a sum aggregation.

Reply:
should we call it `sum_horizontal` instead? and `sum_vertical`? I think we are only dealing with sum aggregations at this step. feel free to propose better names

Reply:
`sum_horizontal` and `sum_vertical` make sense. I could see future use-cases aggregating with weighted average, for example.